staging/src/k8s.io/mount-utils/resizefs_linux.go

```diff
@@ -159,6 +174,22 @@ func (resizefs *ResizeFs) getDeviceSize(devicePath string) (uint64, error) {
 	return size, nil
 }
 
+func (resizefs *ResizeFs) getDeviceRO(devicePath string) (bool, error) {
+	output, err := resizefs.exec.Command(blockDev, "--getro", devicePath).CombinedOutput()
+	outStr := strings.TrimSpace(string(output))
+	if err != nil {
+		return false, fmt.Errorf("failed to read size of device %s: %s: %s", devicePath, err, outStr)
```

Comment:
nit: should the log here be `failed to get read-only ....` or something similar?

Reply:
Woops, fixed. Thank you!